menas/src/main/resources/application.properties.template

```diff
@@ -130,8 +126,11 @@ menas.oozie.mavenSparkJobsJarLocation=/za/co/absa/enceladus/spark-jobs/@project.
 #Menas URL for submitted std and conf jobs
 menas.oozie.menasApiURL=http://menasHostname:8080/menas/api
 
-#Mongo address for spline for the submitted jobs
-menas.oozie.splineMongoURL=mongodb://localhost:27017
+#The URL address of Spline for the submitted jobs
```

Comment:
I would get rid of it _Spline_ reference from Oozie code completely. Let's take if from `application.conf`.

Reply:
I also thought of this, but wouldn't it break Oozie integration?